spec/channel_spec.rb

```diff
@@ -108,12 +108,15 @@ def loop_times(times)
   end
 
   it 'can add members' do
-    response = @channel.remove_members([@random_user[:id]])
+    response = @channel.remove_members([@random_users[0][:id], @random_users[1][:id]])
     expect(response['members'].length).to eq 0
 
-    response = @channel.add_members([@random_user[:id]])
-    expect(response['members'].length).to eq 1
-    expect(response['members'][0].fetch('is_moderator', false)).to be false
+    @channel.add_members([@random_users[0][:id]])
+    response = @channel.add_members([@random_users[1][:id]], hide_history: true)
```

Comment:
I purposefully left 2 statements here. The first one to make sure it works fine without additional parameters. The second one to make sure it works WITH it.